packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
+    }
 
     return React.createElement(component, {
       ...fixture,
       ref: onComponentRef,
-    });
+    }, children);
```

Comment:
This needs a [test](https://github.com/react-cosmos/react-cosmos/blob/b3be0680336724bce8830dae2f367df260394364/packages/react-cosmos/src/components/proxies/__tests__/PropsProxy.jsx). Check out how [props are tested](https://github.com/react-cosmos/react-cosmos/blob/b3be0680336724bce8830dae2f367df260394364/packages/react-cosmos/src/components/proxies/__tests__/PropsProxy.jsx#L31-L33). The [children()](http://airbnb.io/enzyme/docs/api/ShallowWrapper/children.html) method from the Enzyme wrapper should do the trick.